Makefile

```diff
@@ -1,37 +1,62 @@
 # Extremely simple HTTP server that responds on port 8000 with a hello message.
 
-DOCKERHUB_ID:=ibmosquito
+DOCKER_HUB_ID ?= ibmosquito
```

Comment:
@Ruchip16 with the change to `DOCKER_HUB_ID` make sure you change it everywhere `DOCKERHUB_ID` is referenced in the Makefile